LondonBritishLibrary/orient/BLorient2082.xml

```diff
@@ -0,0 +1,375 @@
+                                        ወ<persName ref="PRS14462MalkeaKrestos">መልክዓ፡ ክርስቶስ፡</persName> 
+                                        ዘ<placeName ref="LOC000000000000000000000000">መረዶ፤ </placeName>
+                                        <persName ref="PRS14463BahayloZamaryam">በኃይሎ፡ ዘማርያም፡ </persName>
+                                        ዘ<placeName ref="LOC0000000000000000000000">እምባጭ፡ አርዋ፤</placeName>
+                                        <persName ref="PRS0000000000000000000"><roleName type="title">ማዮቸ፡</roleName> ብሌኖ፤</persName>
+                                        <placeName ref="LOC3549Gojjam">ጕዣሜ፤</placeName>
+                                        አዝማች፡ <persName ref="PRS14464WaldaTansae"><roleName type="title">አባ፡ </roleName>ወልደ፡ ተንሣኤ፤</persName>
+                                        <persName ref="PRS0000000000000000000">ሩፋኤል፤</persName>
+                                        <persName ref="PRS0000000000000000000">ልብሶ፤</persName>
+                                        <persName ref="PRS0000000000000000000"><roleName type="title">በዠር፡ ወንድ፡</roleName> አዕጋረ፡ ክርስቶስ፤</persName>
+                                        <persName ref="PRS0000000000000000000">ተኮ፤</persName>
+                                        <persName ref="PRS14465Elyanos">ኤልያኖስ፤</persName>
+                                        <persName ref="PRS0000000000000000000">እዶ፤</persName>
+                                        <persName ref="PRS0000000000000000000">ተኽሌ፤</persName>
+                                        <persName ref="PRS0000000000000000000">መልኮ። ።</persName>
+                                        በ፲፡ ዓመተ፡ መንግሥቱ፡ ለንጉሥነ፡ <persName ref="PRS4015Fasilada">ፋሲለደስ፡</persName> እንዘ፡ ሀሎ፡
+                                        በ<placeName ref="LOC3577Gondar">ጕንደር።</placeName>
+                                        ዘወሀበት፡ <persName role="donor" ref="PRS14454EhataKrestos"><roleName type="title">ወይዘሮ፡</roleName> 
+                                        እኅተ፡ ክርስቶስ፡</persName> ለመቅደስ፡ <placeName ref="LOC00000000000000000000">ጊዮርጊስ፡ ዘማንጽሩ፡ </placeName> 
+                                        ዘሠረቆ፡ ወዘተአገሎ፡ ወዘአውጽኦ፡ እምደብር፡ እመሂ፡ ንጉሥ፡ አው፡ መኰንን፡ አው፡ በአለ፡ ሥልጣን፡ ውጉዘ፡ ይኩን፡ በአፈ፡ ጴጥሮስ፡ ወጳውሎስ፡ ወበአፈ፡ 
+                                        እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እስከ፡ ለዓለም። ። ። </q>
+                                </colophon>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf">159</measure>
+                                        <measure unit="page">2</measure><locus target="#130v #154v"/>
```

Comment:
I usually do "leaf", because the rest is also calculated/referred to in leaves.

Reply:
Dorothea advised me to use "page" for those blank pages, that are blank only on one side.

Reply:
Denis is right. using `unit="page"` will tell the system that the manuscript is paginated https://betamasaheft.eu/Guidelines/?q=leaf&id=referencing-the-manuscript
Use only "leaf"

Reply:
Ok. I think, it happened also in various other BL manuscript records. I will check.